docs/AxeService.md

```diff
@@ -0,0 +1,80 @@
+# Axe Service
+Terra toolkit automatically includes the wdio-axe-service which enhances an webdriver.io instance with commands for accessibility testing using the [Axe](https://github.com/dequelabs/axe-core) utility.
+
+## Options
+
+Under the key `axe` in the wdio.conf.js you can pass a configuration object with the following structure:
+
+* **inject** - True if the axe script should be injected by the test running. Disable if axe is already included in the test files which slightly speed up runs. Defaults to true.
+
+```js
+// wdio.conf.js
+const wdioConf = require('terra-toolkit/wdio/conf');
+const localIP = require('ip');
+
+const port = 4567;
+
+const config = {
+  ...wdioConf.config,
+
+  // Point base URL to the site to be tested
+  baseUrl: `http://${localIP.address()}:${port}`,
+
+  // Configuration for Axe service
+  axe: {
+    // Don't inject axe script, its included in test files
+    inject: false,
+  },
+};
+
+exports.config = config;
+```
+
+
+## Writing Tests
+
+`browser.axe([{options}]);`
+
+The following options are available:
+
+* **viewports**:
+  An array of viewports `{ width, height }` to run the accessibility test in. If none provided uses the current viewport.
```

Comment:
nit: missing noun in last sentence,
'If none provided, {noun} uses the current viewport.'